src/app/components/main/list/list-item/list-item.component.scss

```diff
@@ -0,0 +1,37 @@
+.listItem {
+  padding-top: 10px;
+  padding-bottom: 10px;
+
+  &__icon {
+    transform: scale(1.8);
+  }
+
+  &__btn {
+    background-color: inherit;
+    cursor: pointer;
+    padding: 2px 3px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-size: 19px;
+    border-radius: 6px;
+  }
```

Comment:
create mixin button or add to general css/scss/less file

Reply:
Done